### PR TITLE
Fixes default_avatar_url method

### LINKED ISSUE #178: 500 error after user registration
Newly registered users will hit a 500 error on https://poster.jellypbc.com after registration. There is no route for the user to log out. 

image

To reproduce the issue, go to https://poster.jellypbc.com/supersecretinvitelink and create a new user account. Once registered, you will hit a 500 error.

I am looking into a fix right now.

### PR DESCRIPTION
**Pull request by cindywu**
fixes #178

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -65,7 +65,7 @@ def avatar_url(variant = nil)
     end
   end
 
-  def self.default_avatar_url
+  def default_avatar_url
```
Comment (dennyluan) [inline app/models/user.rb:68]: This is meant to be a class method, but maybe we could move it to Users Helper instead.

Comment (dennyluan) [reply app/models/user.rb:68]: The change we want is to turn line 64 to

```ruby
User.default_avatar_url
```

instead of `self.default_avatar_url`

**Comment by cindywu** [comment]
I ran into few bumps in the road, but all bumps should be resolved.

- I replaced all instances where we refer to comment text as ```text``` to ```comment```. Our comment model was revised to use ```comment.comment``` instead of ```comment.text```. I made the assumption that when referencing comment text the right name is ```comment``` and not ```text```.

File: app/models/user.rb (hunk 1 above)
Comment (cindywu) [reply app/models/user.rb]: I turned line 64 into ```User.default_avatar_url``` and returned line 68 to the original state. 

I did not move the ```default_avatar_url``` method into Users Helper.

**Comment by cindywu** [comment]
🤔 i had the suspicion that it should be ```text``` let me try to fix

**Comment by cindywu** [comment]
This branch got too messy, so closing. I create a clean PR here https://github.com/jellypbc/poster/pull/182.

**Pull request closed without merging**